rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,184 @@
+  - The TC MUST deliver a decision within one month, via the mechanism described below.
+  - The TC Secretary MUST announce the decision and reasoning for it on the issue in question and remove the `tc-hold` tag.
+  - The decision of the TC is binding.  That means a relevant change MUST NOT be merged until and unless it conforms with the TC's decision.  Depending on the decision, the core developer involved may merge the issue, close it, or revise it accordingly.
+
+If an issue has been marked `tc-hold` for more than two weeks with no response or acknowledgement from the TC, the
+developer SHOULD post to the Internals mailing list a notification of absence.
+
+If a notification of absence receives no response from the TC or any member of it for one month, then the entire TC
+is considered AWOL and removed.  New elections must then be immediately called as described above.
+
+The election of a new TC will "reset the clock" on any outstanding `tc-hold` issues.
+
+==== Decision process ====
+
+=== User-facing changes ===
+
+The RFC process will continue to be used for user facing changes.  However, it will no longer be used for
+non-user-facing technical decisions.
+
+TC members have no special role or authority in the RFC process.
+
+=== Process changes ===
+
+The RFC process will continue to be used for process changes, including changes to the role of the TC.
+
+=== Technical changes ===
+
+As noted, the TC has no special role or authority on user-facing changes.  The TC may, however, decide on the
+implementation of those features. In other words the TC can decide about implementation and ABI but not about API.
+
+The TC SHOULD NOT block the merging of a user-facing change approved by the RFC process, unless the provided
+implementation would result in introduction of new bugs, side effects not mentioned in the RFC, significant performance
+penalties, or if there is an equivalent implementation in progress that TC finds more appropriate.
+
+The TC SHOULD give due consideration and weight to previous decisions regarding technical aspects, either by the TC
+or by RFCs prior to the TC's creation.  However, it is NOT REQUIRED to abide by them, and may make decisions as it
+sees best to ensure the quality and maintainability of the code at the time the decision is made.
+
+Technical aspects on which the TC may make a decision include, but are not limited to:
+
+  * Whether a given PR is a feature or a bug.
+  * Whether a given change qualifies as user-facing, and thus requires an RFC.
+  * Whether a given technical aspect or implementation approach is acceptable.
+
+=== TC voting ===
+
+Once an issue has been brought to the TC, the Secretary must allow all TC members time to offer their input within one
+week.  Once the Secretary feels the issue has been sufficiently discussed, the Secretary will call a vote of the TC
+members. The vote may be public or private and conducted however the TC chooses.
```

Comment:
public or private? I would pick just one. I think `private` is more appropriate here.

Reply:
Yeah we can probably just remove `public` as I wouldn't assume that it would be public anyway.

Reply:
Removed public.